client/src/extension.ts

```diff
@@ -80,36 +61,39 @@ function startLangServer(
 }
 
 export function activate(context: ExtensionContext): void {
-    if (context.extensionMode === ExtensionMode.Development) {
-        const cwd = path.join(__dirname, "..", "..");
-        console.log("Hiasdfasdfasdf");
-        const pythonPath = workspace
-            .getConfiguration("python")
-            .get<string>("defaultInterpreterPath");
+    const cwd = path.join(__dirname, "..", "..");
+    const platform = os.platform();
+    let pythonInterpreterPath = 'python3';
+    if (platform === 'win32') {
```

Comment:
Not an expect, but it looks like we're trying to install python EVERY time the extension activates?